### DISCUSSION TITLE
Resource for identifying "variable features" from the reducedDim object

### DISCUSSION THREAD

**Discussion by janekim319** [GreenleafLab/ArchR#957] [Questions / Documentation]
Hi Ryan,

Thank you very much for your kind answers in my previous questions. I'm wondering if you or really anyone could direct me to the resources that can help me identify the variable bins/features from the reducedDim object created from addIterativeLSI(). I'm trying to see if adding iterations can help identify sub populations better, but am having trouble identifying where the variable features information is stored.

**Comment by janekim319** [comment]
Or perhaps a way to get out the bins that ArchR retained after iterativeLSI?

**Comment by rcorces** [comment]
`addIterativeLSI()` has an option called `saveIterations` which, when set to `TRUE` will save RDS objects of the individual LSI iterations in a subfolder of the ArchRProject output directory. I would start there to see if that gives what you are looking for. The output folder should have the name of whatever you called your reducedDims object.
https://www.archrproject.com/reference/addIterativeLSI.html
